common/src/main/java/xyz/jpenilla/minimotd/common/config/MiniMOTDConfig.java

```diff
@@ -141,6 +141,10 @@ public MOTD(final @NonNull String line1, final @NonNull String line2) {
         + "\n"
```

Comment:
GitHub won't allow me to comment on the line above this one, however the line above this one needs to be updated to include `constant` and `minimum`.